gpbft/options.go

```diff
@@ -97,3 +102,32 @@ func WithMaxLookaheadRounds(r uint64) Option {
 		return nil
 	}
 }
+
+// WithRebroadcastBackoff sets the duration after the gPBFT timeout has elapsed, at
+// which all messages in the current round are rebroadcast if the round cannot be
+// terminated, i.e. a strong quorum of senders has not yet been achieved.
+//
+// The backoff duration grows exponentially up to the configured max. Defaults to
+// exponent of 1.3 with 3s base backoff growing to a maximum of 30s.
```

Comment:
Nit: The comment about the defaults will likely rot since they're defined 100 lines away.

Reply:
I moved the default var definition right above the option.